### PR TITLE
Add doc install instructions for non-root users.  Addresses #2034.

### PR DESCRIPTION
**Pull request by DailyDreaming**
Provide instructions for installing toil without root privileges, as per @mr-c 's instructions.

Thank you @mr-c !

### CODE REVIEW

**Comment by DailyDreaming** [comment]
@diekhans I changed it up, let me know what you think!

File: docs/gettingStarted/install.rst [hunk 1]
```diff
@@ -5,40 +5,36 @@
...
 -----------------------------------------
 
-Toil currently  supports only Python 2.7.  If you don't satisfy this requirement, consider using anaconda_ to create
-an alternate Python 2.7 installation.
+Toil currently supports only Python 2.7 and requires a virtualenv to be active to install.
 
-.. _anaconda: https://conda.io/docs/py2or3.html 
-
-Install Python ``virtualenv`` using pip_.
+First, install the latest Python ``virtualenv`` using pip_.
```
Comment (diekhans) [inline docs/gettingStarted/install.rst]: many people will already have virtualenv installed. maybe; 

> If not already present,  install the latest Python ``virtualenv`

**Comment by diekhans** [comment]
Excellent, I made one minor suggestion.

**Comment by DailyDreaming** [comment]
I've updated based on your suggestion!  Thanks @diekhans !

**Comment by DailyDreaming** [comment]
Flaky test: src.toil.test.jobStores.jobStoreTest.AWSJobStoreTest.testCleanCache

**Comment by DailyDreaming** [comment]
Jenkins, test this please.

**Pull request merged**